APIs/schemas/input.json

```
@@ -21,7 +22,15 @@
           "description": "Whether the Input supports EDID",
           "type": "boolean"
         },
-        "effective_edid": { "$ref": "edid.json" }
+        "effective_edid": { "$ref": "edid.json" },
+        "signal_state": {
+          "type": "string",
+          "enum": [
+            "no_signal",
+            "awaiting_signal",
+            "signal_is_present"
```

Or `signal_stable` ?

```suggestion
            "signal_present"
```

---

IS-11 must work without Inputs. This state relates to the transitional period when an upstream device changes its signal (resolution, grain rate, etc). During the reacquisition of the signal a device would indicate awaiting_essence.  The state no_essence indicates that there us no signal at all …

---

Separating the state of an Input from the state of a Sender doesn't prevent us from having IS-11 without Inputs. `awaiting_signal` for an Input makes sense for the case you've described. The question is whether `awaiting_essence` makes sense for a Sender. As far as there may be a Sender completely depending from its Inputs, I think it makes sense. From the client's perspective PUTting Active Constraints causes the reacquisition of the signal and therefore the reacquisition of the Essence in this case.